File: app/src/main/java/il/ronmad/speedruntimer/GamesAdapter.java
package il.ronmad.speedruntimer;

import android.content.Context;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import java.util.List;

public class GamesAdapter extends MyBaseAdapter<Game> {

    public GamesAdapter(Context context, List<Game> games) {
        super(context, games);
        this.listItemResourceId = R.layout.games_list_item;
    }

    @Override
    public View getView(int i, View view, ViewGroup viewGroup) {
        View listItemView = super.getView(i, view, viewGroup);

        TextView text = listItemView.findViewById(R.id.gameName);
        text.setText((getItem(i)).name);

        return listItemView;
    }
}

File: app/src/main/kotlin/il/ronmad/speedruntimer/GamesAdapter.kt
package il.ronmad.speedruntimer

import android.content.Context
import android.view.View
import android.view.ViewGroup
import kotlinx.android.synthetic.main.games_list_item.view.*

class GamesAdapter(context: Context, games: List<Game>)
    : MyBaseListFragmentAdapter<Game>(context, games, R.layout.games_list_item) {

    override fun getView(i: Int, view: View?, viewGroup: ViewGroup): View {
        val listItemView = super.getView(i, view, viewGroup)

        listItemView.gameName.text = getItem(i).name
        return listItemView
    }
}
